src/components/panel/panel.js

```diff
@@ -1188,6 +1211,24 @@ MdPanelRef.prototype._addStyles = function() {
 
 
 /**
+ * Update the position configuration of a panel
```

Comment:
nit: Method descriptions should start with a sentence written in the third person declarative voice. Can you change this to:

"Updates the position configuration of the panel."